### ISSUE TITLE
[FEATURE] - Compare owned Plex Library with other libraries to determine missing media

### ISSUE THREAD

**Issue by alaska4life** [Reaparr/Reaparr#73]
A nice feature would be an automated way to compare the movies in your library to the movies in a remote library, so you can just see the ones that you don't have and select them for download.

**Comment by JasonLandbridge** [comment]
Yeah that is a big feature I would like as well! I'm not sure how that would look however, how would you like to see it? When you see a movie in a different library you don't have, should it have a green glowing border if its new and a yellow border if this is a higher quality one? Or a filter which only shows new or higher quality movies? Or everything on a different page?

I'm also planning to make the home page which now says "Working" a collection of all Media over all the different servers so you can just keep scrolling on 1 page to see everything. 

Anyway please let me know if you have ideas!

**Comment by alaska4life** [comment]
Maybe a "Missing" tag and a "Higher Quality" tag.  Another thing that would help would be to put sort the movies in the same order as plex.  For instance, the movie "A Beautiful Day in the Neighborhood" sorts at the top of the A's in PlexRipper, but sorts after "Bea" movies in the B section in Plex.  Makes it difficult to compare libraries.

**Comment by JasonLandbridge** [comment]
I've added this as a goal for release v0.11, other things take some priority but I'm still a huge fan of this idea!

**Comment by i2sly** [comment]
just figured i would jump in and say that this would be an awesome option to have and for someone like me an additional option to grab all that is not on my server would be great

**Comment by tshirtnjeans** [comment]
I've been hoping for this feature for a while. Where are we on any progress with this?

I usually just have my Plex library open side by side with Reaparr and scroll them both at the same time. Perhaps a simple toggle that hides titles in other libraries that are already in your library?

**Comment by JasonLandbridge** [comment]
@tshirtnjeans Yeah me too! This is the last big feature I want to implement before releasing 1.0 so I will most likely work on this next. It is quite complex in terms of how I want to display this in the UI, since you can have missing episodes, or 1 episode is higher quality etc.

**Comment by JasonLandbridge** [comment]
Coming soon!

<img width="2591" height="1603" alt="Image" src="https://github.com/user-attachments/assets/c8a20838-1b47-40ab-b5fe-d745726771c8" />

I'm struggling with coming up with ideas on how best to show what is missing or where a higher quality media item is available, any ideas? The icons I currently have are hard to see when overlaid on a media poster

**Comment by isotisot** [comment]
It looks like the highlighted box will filter the output?  That would be ideal since you could filter out movies that were already owned or higher quality and just see movies that are missing or that you have lower quality of.

**Comment by JasonLandbridge** [comment]
@isotisot 

> It looks like the highlighted box will filter the output? That would be ideal since you could filter out movies that were already owned or higher quality and just see movies that are missing or that you have lower quality of.

Yes indeed, so that is already working and you are also able to combine it with other filters such as "country" or "genre" or search through the filtered compared media as normal. Before this feature I've also greatly optimized the home page which shows a collection of ALL media based on Movies or TvShows and these filters will work there too. So with some filtering and sorting you can have the latest updated missing movies and download them all in 1 go.

I'm also going to add a feature where you are able to hide/disable specific Plex libraries which contain content you are never interested in, such as Bollywood media or work-out videos etc, this way you can see and go through the media you are more likely to be interested in

**Comment by JasonLandbridge** [comment]
This feature has been released in the latest 0.39 version!
